lib/jelix-admin-modules/jacl2db_admin/controllers/users.classic.php

```diff
@@ -61,6 +59,7 @@ public function index()
         $offset = $this->param('idx', 0, true);
         $grpid = $this->param('grpid', jAcl2DbAdminUIManager::FILTER_GROUP_ALL_USERS, true);
         $filter = trim($this->param('filter'));
+        $last = count($groups) - 3;
```

Comment:
why `-3`? I guess it should be `-1` right?

Reply:
`$groups` is pre-populated with 2 groups that will never be displayed, lines 43 and 49.

Reply:
ok I see